modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/provider.go

```diff
@@ -192,3 +169,8 @@ func (p *CustomFilter) RegisterFilterItemDeleteOp(opData filter.OpFilterItemDele
 		return nil
 	}
 }
+
+func (p *CustomFilter) setDefaultValues() {
+	p.State.FrontendConditionValues.App = []string{common.Participated}
+	p.State.FrontendConditionValues.Branch = []string{common.DefaultBranch}
```

Comment:
there are two default branch existing, master and develop.

before we allow user to config project-level default branch, we should put them both as default.